content/blog/environments-secrets-configurations-management/index.md

```diff
@@ -87,7 +87,7 @@ Pulumi ESC was born to address these problems and needs head on.  It does so thr
 * __Auditable__: Environments must be "opened" to compute and see the set of value they provide, and this action is recorded in audit logs, including a full record of how each value was sourced from within the hierarchy of environments that contributed to it.
 * __Authentication and RBAC__:  Pulumi ESC brokers access to secrets and configuration that live in other systems, and so authentication and granular RBAC are critical to ensure robust access controls across your organization.  Pulumi ESC leverages the same Pulumi Cloud identity, RBAC, Teams, SAML/SCIM and scoped access tokens that are used for Pulumi IaC today, extending these all to managing access to Environments as well as Stacks.
 * __Configuration as Code__:  Environments are defined as YAML documents which can describe how to project and compose secrets and configuration, integrate dynamic configuration providers, and compute new configuration from other values (construing a URL from a DNS name, or concatenating multiple configuration values into a derived value).  The incredible flexibility of a code-based approach over traditional point-and-click interfaces allows Pulumi ESC to offer rich expressiveness for managing complex configuration.
-* __Fully Managed__: Pulumi ESC is offered as a fully managed cloud service in Pulumi Cloud (and Pulumi Cloud Self-hosted in the near future). The pulumi/esc project is [open source](https://github.com/pulumi/esc/), and contains the evaluation engine for environments, the esc CLI, and in the future, the extensible plugins for source and target integrations.
+* __Fully Managed__: Pulumi ESC is offered as a fully managed cloud service in Pulumi Cloud. The pulumi/esc project is [open source](https://github.com/pulumi/esc/), and contains the evaluation engine for environments, the esc CLI, and in the future, the extensible plugins for source and target integrations.
```

Comment:
@arunkumar611 Do we have a preferred format for updating blog posts?

Reply:
"Pulumi ESC is offered both as a fully managed and self-hosted service in Pulumi Cloud"